yaml/xpu_functions.yaml

```diff
@@ -386,6 +386,10 @@ supported:
   - acosh.out
   - addr
   - addr.out
+  - avg_pool2d
```

Comment:
Will this get `aten::_adaptive_avg_pool2d` supported? This one is required for efficientnet inference.

Reply:
we will have another pr for this op